### ISSUE TITLE
Boot interpreting error message as format string.

### ISSUE THREAD

**Issue by au-phiware** [degree9/boot-exec#18]
For whatever reason the process I try to invoke returns an error message that contains `%2f` and subsequently I receive this error from boot-exec:

```
Executing Process: /usr/local/bin/npm
Executing Process with arguments: ["install" "--production"]
clojure.lang.ExceptionInfo: Format specifier '%2f' {:line 243}
	at clojure.core$ex_info.invokeStatic(core.clj:4617)
	at clojure.core$ex_info.invoke(core.clj:4617)
	at boot.main$_main$fn__1201.invoke(main.clj:222)
	at boot.main$_main.invoke(main.clj:216)
	at clojure.lang.Var.invoke(Var.java:394)
	at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:159)
	at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:150)
	at boot.App.runBoot(App.java:399)
	at boot.App.main(App.java:491)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at Boot.main(Boot.java:257)
Caused by: java.util.MissingFormatArgumentException: Format specifier '%2f'
	at java.util.Formatter.format(Formatter.java:2519)
	at java.util.Formatter.format(Formatter.java:2455)
	at java.lang.String.format(String.java:2940)
	at clojure.core$format.invokeStatic(core.clj:5533)
	at clojure.core$format.doInvoke(core.clj:5527)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:646)
	at clojure.core$apply.invoke(core.clj:641)
	at boot.util$print_STAR_.invoke(util.clj:75)
	at boot.util$info.doInvoke(util.clj:142)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at degree9.boot_exec$exec_impl.invokeStatic(boot_exec.clj:62)
	at degree9.boot_exec$exec_impl.invoke(boot_exec.clj:49)
	at degree9.boot_exec$eval1365$fn__1366$fn__1371$fn__1372.invoke(boot_exec.clj:79)
	at degree9.boot_exec$eval1322$fn__1323$fn__1328$fn__1329.invoke(boot_exec.clj:30)
	at boot.core$run_tasks.invoke(core.clj:1021)
	at boot.core$boot$fn__933.invoke(core.clj:1031)
	at clojure.core$binding_conveyor_fn$fn__4676.invoke(core.clj:1938)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by au-phiware** [comment]
I think this is fixed in master... checking now.

Nope, not fixed, see #19

**Comment by burn2delete** [comment]
Should now be fixed in master.
